src/display/api.js

```diff
@@ -262,7 +264,8 @@ function getDocument(src, pdfDataRangeTransport,
         throw new Error('Loading aborted');
       }
       var messageHandler = new MessageHandler(docId, workerId, worker.port);
-      var transport = new WorkerTransport(messageHandler, task, rangeTransport);
+      var transport = new WorkerTransport(messageHandler, task, rangeTransport,
+                                          canvasFactory);
```

Comment:
Based on https://github.com/mozilla/pdf.js/pull/8002#discussion_r99022111, this parameter should be removed.